src/thunderbots/software/backend_output/backend.h

```diff
@@ -10,8 +10,8 @@ class Backend
      *
      * @param primitives the list of primitives to send
      */
-    virtual void sendPrimitives(
-        const std::vector<std::unique_ptr<Primitive>> &primitives) = 0;
+    virtual void sendPrimitives(const std::vector<std::unique_ptr<Primitive>>& primitives,
+                                Team& team) = 0;
```

Comment:
```suggestion
                                const Team& team) = 0;
```
(will also need to be changed in the `.cpp`, I'm just testing this new-fangled "suggestion" feature out)